layout-server/src/main/java/org/finos/vuu/layoutserver/dto/response/MetadataResponseDTO.java

```diff
@@ -0,0 +1,18 @@
+package org.finos.vuu.layoutserver.dto.response;
+
+import java.util.Date;
+import java.util.UUID;
+import lombok.Data;
+import org.finos.vuu.layoutserver.dto.MetadataDTO;
+
+@Data
+public class MetadataResponseDTO implements MetadataDTO {
+
+    private UUID layoutId;
+    private String name;
+    private String group;
+    private String screenshot;
```

Comment:
is this the encoded image ? Does requesting a list of available layouts always return the images with the response ? I'm wondering if we might also want a lightyer weight alternative, for populating dropdown list etc.
Just a thought, not a requirement right now.

Reply:
Couple of thoughts to this one.

In terms of the screenshot string, we've got a spike ticket in our backlog to explore alternative means of saving the screenshot PNG, as the current base64 encoded string method isn't ideal. It's a minimal file size, but as you say, the length of the string itself is a lot to be sending between client and server and persisting in the database.

In terms of modifying the response itself, we can add an additional endpoint with a new DTO for different client-side requirements as and when they crop up (like fetching data for the dropdown list).